### DISCUSSION TITLE
ERROR: OS file watch limit reached.

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#5453] [Q&A]
I get this error when I am previewing a quarto website. It doesn't happen all the time. It seems a bit random. And I am not quite sure how to reproduce it. The preview still works, so I don't know if this is really a problem.

```
ERROR: OS file watch limit reached. about ["/home/user/Documents/website/site_libs"]

Error: OS file watch limit reached. about ["/home/user/Documents/website/site_libs"]
    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)
    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)
    at runWatcher (file:///opt/quarto/bin/quarto.js:102953:30)
    at file:///opt/quarto/bin/quarto.js:102964:45
    at Array.forEach (<anonymous>)
    at watchForChanges (file:///opt/quarto/bin/quarto.js:102958:37)

```

```
🢒  quarto --version
1.3.340
```

**Comment by mcanouil** [comment]
Could you share a preferably small working (reproducible) example to work with? Thanks.
The example could be Git repository.

Without any means to reproduce, it's unlikely a fix can be found.

**Comment by mazzma12** [comment]
From my understanding this is due to your OS, and likely another program having a lot of activities at the same time. This happened to me with VSCode, closing it allowed to preview
